## Limpieza de datos
Para llevar a cabo la limpieza de dato seguiremos un proceso sistemático que aborde problemas tanto observados en la exploración como problemas comunes en los datos crudos. La limpieza de datos es crucial, ya que garantiza que el análisis posterior sea preciso y significativo.

### Importación de librerias

In [1]:
import pandas as pd
import numpy as np

### Carga del dataframe

In [2]:
df = pd.read_csv("../data/processed/combined_data.csv")

## Identificación de inconsistencias

In [3]:
# Valores nulos por columna
df.isnull().sum()

Unnamed: 0                             0
id                                     0
Gender                                 0
Customer Type                          0
Age                                    0
Type of Travel                         0
Class                                  0
Flight Distance                        0
Inflight wifi service                  0
Departure/Arrival time convenient      0
Ease of Online booking                 0
Gate location                          0
Food and drink                         0
Online boarding                        0
Seat comfort                           0
Inflight entertainment                 0
On-board service                       0
Leg room service                       0
Baggage handling                       0
Checkin service                        0
Inflight service                       0
Cleanliness                            0
Departure Delay in Minutes             0
Arrival Delay in Minutes             393
satisfaction    

**Observación:** Encontramos un total de 393 datos nulos, todos ellos en la columna "Arrival Delay in Minutes" representado aproximadamente un 0.30% del total de filas de nuestro Dataframe.